mindsdb/api/mysql/mysql_proxy/datahub/datanodes/mindsdb_datanode.py

```diff
@@ -127,10 +127,7 @@ def select(self, table, columns=None, where=None, where_data=None, order_by=None
             for key in min_max_keys:
                 row[key + '_min'] = res[i].explanation[key]['confidence_interval'][0]
                 row[key + '_max'] = res[i].explanation[key]['confidence_interval'][-1]
+            row['select_data_query'] = select_data_query
```

Comment:
Potentially good to bring back the check here for when we support `external_data_source` instead of `select_data_query` ?

Reply:
`external_data_source` instead of `select_data_query` ? i think we should use both, or not?

If talk about `select_data_query` than it will be better do here so here:
```
    if came_from is not None:
        for row in data:
            row['select_data_query'] = select_data_query
```
but: above is method `getTableColumns`, and it returns for each predictor table column `select_data_query` in list. That not  very good, because this column will be return in plain sql query to mindsdb too. That need to fix in future, but for now we must set `select_data_query` in each row.